This file shows how to run shlibsign under dbx, the Sun Studio debugger, to see
where it's aborting.

--- nss-3.12.4-with-nspr-4.8/mozilla/security/nss/cmd/shlibsign/./sign.sh.orig	2009-11-22 15:17:12.069116849 +0100
+++ nss-3.12.4-with-nspr-4.8/mozilla/security/nss/cmd/shlibsign/./sign.sh	2009-11-22 15:17:54.559576926 +0100
@@ -34,7 +34,7 @@
     export LIBRARY_PATH
     ADDON_PATH=${1}/lib:${4}:$ADDON_PATH
     export ADDON_PATH
-    echo ${2}/shlibsign -v -i ${5}
-    ${2}/shlibsign -v -i ${5}
+    echo /opt/studio/SOS11/SUNWspro/bin/dbx ${2}/shlibsign -v -i ${5}
+    /opt/studio/SOS11/SUNWspro/bin/dbx ${2}/shlibsign -v -i ${5}
     ;;
 esac
